### PR TITLE
Add feed flow

### PR DESCRIPTION
**Pull request by mconnell**
This PR changes the initial flow of creating a new feed to steer the user through the process of selecting content after creating a new feed.

### CODE REVIEW

File: app/views/admin/firehose_feed_sources/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+             builder: Dough::Forms::Builders::Validation do |form| %>
+
+  <%= form.collection_check_boxes :source_ids, @sources, :id, :name do |b| %>
+    <%= form.form_row(:source) do %>
+      <%= b.check_box %>
+      <%= b.label { b.text } %>
+    <% end %>
+  <% end %>
+
+  <div>
```
Comment (benbarnett) [inline app/views/admin/firehose_feed_sources/new.html.erb]: not sure what value this `div` brings?

File: app/views/admin/partners/show.html.erb [hunk 2]
```diff
@@ -19,11 +19,23 @@ Url: <%= link_to @partner.url, @partner.url %>
...
+      <% @partner.feeds.each do |feed| %>
+        <tr>
+          <td><%= feed.name %></td>
+          <td><%= feed.description %></td>
+          <td><%= link_to 'Show feed', admin_partner_feed_path(@partner, feed) %></td>
+        </tr>
+      <% end %>
+    </tbody>
+  </table>
+</div>
```
Comment (benbarnett) [inline app/views/admin/partners/show.html.erb]: Newline....

File: app/views/admin/partners/show.html.erb [hunk 3]
```diff
@@ -19,11 +19,23 @@ Url: <%= link_to @partner.url, @partner.url %>
...
-    <%= link_to 'Show feed', admin_partner_feed_path(@partner, feed) %>
-  </div>
-<% end %>
+<div class="table-wrapper">
+  <table class="datatable-default">
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Description</th>
+        <th colspan="2">Actions</th>
```
Comment (benbarnett) [inline app/views/admin/partners/show.html.erb]: Do we need colspan? Looks like there's only 3 columns below too.

File: app/views/admin/feeds/_form.html.erb [hunk 4]
```diff
@@ -31,7 +31,11 @@
   <% end %>
 
   <%= form.form_row do %>
-    <%= link_to 'cancel', feed.persisted? ? url : [:admin, partner] %>
-    <%= form.submit feed.persisted? ? 'Update feed' : 'Create feed' %>
+    <% if feed.persisted? %>
+      <%= link_to 'cancel', url %>
+      <%=  form.submit 'Update feed' %>
```
Comment (benbarnett) [inline app/views/admin/feeds/_form.html.erb]: Double space.

File: app/views/admin/feeds/_form.html.erb [hunk 5]
```diff
@@ -31,7 +31,11 @@
...
+    <% else %>
+      <%=  form.submit 'Next' %>
```
Comment (benbarnett) [inline app/views/admin/feeds/_form.html.erb]: Double space.

**Comment by mconnell** [comment]
Thanks @benbarnett. Updated accordingly.

**Comment by benbarnett** [comment]
Cool, frontend seems good now :+1:

**Comment by JiggyPete** [comment]
The only thing I can pick up on is that the text in the html files isn't coming from a I18n YAML file?

**Comment by mconnell** [comment]
If we left them inline, we would have started to have multiple places where we had conditional logic appearing in the view. We originally had them split and decided it was easier to read splitting them back out.

**Comment by mconnell** [comment]
We don't require localisation, it's an internal project that only has english users.

**Comment by JiggyPete** [comment]
:+1:

**Pull request merged**